queries/javascript/highlights.scm

```diff
@@ -91,6 +91,10 @@
   (rest_parameter
     (identifier) @parameter))
 
+(variable_declarator                                                                                                                                                                                        
+    value: (arrow_function                                                                                                                                                                                    
+      parameter: (identifier) @parameter))
```

Comment:
I don't think this solves the issue presented in the description. The syntax tree for the expression is

```
            value: arrow_function [37, 10] - [37, 33]
              parameters: formal_parameters [37, 10] - [37, 27]
                identifier [37, 11] - [37, 16]
                object_pattern [37, 18] - [37, 26]
                  shorthand_property_identifier [37, 20] - [37, 24]
```

A) You would need to change it to `parameters`
B) You would need to change `identifier` to `shorthand_property_identifier` to target the destructered property.

Reply:
ive already asked to change the shorthand property as its not a parameter by itself, the object is.

Reply:
he removed it after my previous comment. But i didn't expect it to be removed, i just noticed that a shorthand_property_identifier is not a parameter but an object property.
I don't know if assigning to parameter would be a good idea. What do you think about that ?

Reply:
It's an object property, but since it's only in the context of function parameters it might be ok. Would this be better for a custom query extension on the users end?

Reply:
I added it(object property destructuring) only because I used to it looks like this in Sublime editor, but @kyazdani42 had a good point that it is should be what it is in a first place)))